scripts/run-e2e-test.sh

```diff
@@ -18,29 +18,78 @@ function toggle_windows_scheduling(){
   done
 }
 
-echo "Cordon off windows nodes"
+ROLE_NAME="AWS-LB-CONTROLLER-ROLE"
```

Comment:
I recommend using a role name prefix, and attach the e2e test id as well. This will helps us run multiple tests in parallel.

Reply:
yes will update it

Reply:
Updated